### PR TITLE
Wishlist Management

### PR DESCRIPTION
**Pull request by saicharan-gouru**
-Added wishlist functionality
-Move to wishlist functionality
-Remove from wishlist functionality
-Clear wishlist functionality
-Move to cart functionality
wishlist management

### CODE REVIEW

File: src/components/NavBar/NavBar.jsx [hunk 1]
```diff
@@ -16,8 +17,8 @@ function NavBar(){
                 <Link to="/products" className="nav-link hover-effect" onClick={()=>productsDispatch({type:"CLEAR"})}>Grocery</Link>
                 <input className="search-bar" autoFocus/>
                 <Link to="/login" className="nav-link hover-effect">Login/Signup</Link>
-                <Link to="/cart" ><ShoppingCartIcon className="nav-link hover-effect" style={{ fontSize: "30px" }}></ShoppingCartIcon> <span className="cart-badge"> {items_count} </span> </Link>
-                <Link to="/wishlist" ><FavoriteIcon className="nav-link hover-effect" style={{ fontSize: "30px" }}></FavoriteIcon></Link>
+                <Link to="/cart" ><ShoppingCartIcon className="nav-link hover-effect" style={{ fontSize: "30px" }}></ShoppingCartIcon> <span className="badge"> {items_count} </span> </Link>
```
Comment (ianuragsharma) [inline src/components/NavBar/NavBar.jsx:20]: avoid inline style

File: src/pages/cartpage/components/CartProductCard.js [hunk 2]
```diff
@@ -31,7 +33,11 @@ function CartProductCard({product}){
 
                             </div>
                             <button className="button primary-blue" onClick={()=>cartDispatch({type:"REMOVE_FROM_CART",payload:product})}>Remove from cart</button>
-                            <button className="button primary-blue">Move to wishlist</button>
+                            { 
+                            itemsInWishlist.some(item => item._id === _id) ? 
+                            <Link to="/wishlist"><button className="button primary-yellow">Go to wishlist</button></Link> :
+                            <button className="button primary-blue" onClick={()=>wishlistDispatch({type:"MOVE_TO_WISHLIST",payload:product})}>Move to wishlist</button>
```
Comment (ianuragsharma) [inline src/pages/cartpage/components/CartProductCard.js:39]: i would suggest not to clutter your jsx with logic you can extract some part from it and implement it accordingly.

**Review by ianuragsharma — APPROVED** [review]
everything look fine to me nothing code breaking.I have left some sanity comments
Please consider the changes i have suggested.

**Review by hrishikeshmane — APPROVED** [review]
Looks good. Just format your code with prettier before pushing.

**Pull request merged**